deploy/role.yaml

```diff
@@ -14,6 +14,13 @@ rules:
   - events
   - configmaps
   - secrets
+  - serviceaccounts
```

Comment:
All new CR/CR and OCP types which will be used by it need to be added. 
I hope that the following points help you with. 
- https://github.com/aerogear/mobile-security-service-operator/blob/master/deploy/cluster_role.yaml#L28
- https://github.com/aerogear/mobile-security-service-operator/blob/master/deploy/cluster_role.yaml#L46

Reply:
You're right that maybe `oauthclients` is needed here.

I believe the CR is already covered here from before this PR, in another part of the file ([here](https://github.com/aerogear/mobile-developer-console-operator/blob/bb3eb737461b60528a16d1a3cc4eb895e44fb001/deploy/role.yaml#L56..L61)).

Reply:
We don't create/list/get OAuthClients in the operator.

I know I told you I will implement that, but later we had a meeting with @odra and decided to simplify things.

Quoting the PR description:
>> Instead of creating / patching the OAuthClient in the operator, we will take the approach done in the RHMI webapp.
>> Ansible will install the operator, it will create a CR to provision MDC server, then it will patch the OAuthClient with the route of the MDC server.

Sorry for the confusion folks

Reply:
:+1: